baggage_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otel
+
+import (
+	"context"
+	"testing"
+
+	"go.opentelemetry.io/otel/internal/baggage"
+	"go.opentelemetry.io/otel/label"
+)
+
+func TestBaggage(t *testing.T) {
+	ctx := context.Background()
+	ctx = baggage.ContextWithMap(ctx, baggage.NewEmptyMap())
+
+	b := Baggage(ctx)
+	if b.Len() != 0 {
+		t.Fatalf("empty baggage returned a set with %d elements", b.Len())
+	}
+
```

Comment:
Shall we add a `Baggage(ctx)` with some map values to cover `baggage.go#L32-L34`